src/main/resources/config/application-buildagent.yml

```diff
@@ -0,0 +1,42 @@
+# ===================================================================
+# Build Agent specific properties: this file will only be loaded during startup if the profile buildagent is active
+#
+# This configuration overrides the application.yml file.
+# ===================================================================
+artemis:
+    repo-clone-path: ./repos
+    repo-download-clone-path: ./repos-download
+    version-control:
+        url: https://artemislocal.cit.tum.de  # TODO: is this really needed for build agents?
+        user: "demo"                    # not needed for build agents / TODO: remove this requirement in GitService
+        password: "demo"                # not needed for build agents / TODO: remove this requirement in GitService
```

Comment:
Yes. We haven't fully decided what the yml will look like. This is just a first version so that the build agent can run. We will decide on a more definite version in a follow up